crates/tauri-plugin-client/android/settings.gradle

```diff
@@ -11,7 +11,7 @@ pluginManagement {
         eachPlugin {
             switch (requested.id.id) {
                 case "com.android.library":
-                    useVersion("8.8.0")
+                    useVersion("8.7.3")
```

Comment:
Pin android gradle plugin for compatability with NDK version included by nix flake